Backend/src/test/java/Digitact/Backend/Model/EducationTest.java

```diff
@@ -0,0 +1,13 @@
+package Digitact.Backend.Model;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class EducationTest {
+
+    @Test
+    void createEducation() {
+        Education education = new Education();
+        Assertions.assertThat(education).isNotNull();
```

Comment:
May I ask in which cases this assertion would fail?

Reply:
These tests in Model classes are highly unlikely to fail.